src/parts/Form/index.jsx

```diff
@@ -189,7 +168,7 @@ const Form = () => {
     // NOTE • Start the node
     const startNode = async () => {
         if (!bootnodes || bootnodes.length === 0) {
-            alert('Genesis hash and at least one bootnode are required.');
+            console.error('Genesis hash and at least one bootnode are required.');
```

Comment:
I think alert should stay, as it presents error to the user

Reply:
I mean, likely they did, but console error is not a best replacement, it doesn't give any feedback for a user without a console open